package.json

```diff
@@ -107,17 +109,17 @@
     "z-index": "0.0.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.14.5",
-    "@babel/core": "^7.14.6",
-    "@babel/node": "^7.14.7",
-    "@babel/preset-env": "^7.14.7",
-    "@babel/preset-react": "^7.14.5",
-    "@babel/preset-typescript": "^7.17.12",
+    "@babel/cli": "7.14.5",
```

Comment:
I'm not totally sure why `^` need to be removed from a lot of packages here. I'm guessing you encountered some kind of dependency conflicts is that it?

Reply:
I can remove these. They were just causing issues with the install of metabase.

Reply:
Then can you please add `^` back?

Reply:
The ^s have been added back.